frontend/src/components/crud/DisplayTableContainer.tsx

```diff
@@ -68,9 +71,22 @@ const columns: Column<EntityData>[] = [
 
     accessor: "boolField",
   },
+  {
+    Header: "fileName",
+
+    accessor: "fileName",
+
+    // eslint-disable-next-line react/display-name
+    Cell: ({ cell }: any) =>
+      cell.row.values.fileName ? (
+        <button type="button" onClick={downloadEntityFile}>
```

Comment:
As mentioned earlier :)

```suggestion
        <button type="button" onClick={() => downloadEntityFile(cell.row.values.fileName)}>
```